lib/css/components/toggle-switches.less

```diff
@@ -5,6 +5,14 @@
     --_ts-multiple-bg: unset;
     --_ts-multiple-fc: var(--black-500);
 
+    fieldset[disabled] & {
+        &,
+        & label {
```

Comment:
Optional: simplification? removes a level of indentation, but don't know if it is cleaner

```
    fieldset[disabled] &, fieldset[disabled] & label {
```

Reply:
I honestly prefer what is now in this case, but I think we'd be wise to look out for ugly nested `&`. I've been bit by misinterpretation there and I'll keep in mind where this style might be a little more confusing than it's worth.